frontend/talentsearch/src/js/types/PoolUtils.ts

```diff
@@ -0,0 +1,8 @@
+import { Classification } from "@common/api/generated";
+
+export type SimpleClassification = Pick<Classification, "group" | "level">;
+
+export type SimplePool = {
+  id: string;
+  classification?: SimpleClassification;
```

Comment:
The type Pool has `classifications` (plural) but SimplePool has a single `classification`. That could could some confusion, and potentially bugs, I think.